components/hoc/Portal.js

```diff
@@ -68,7 +68,7 @@ class Portal extends Component {
 
   _unrenderOverlay () {
     if (this._overlayTarget) {
-      if (this.props.lockBody) document.body.style.overflow = 'scroll';
+      if (this.props.lockBody) document.body.style.overflow = null;
```

Comment:
The problem is because it is an inline style, 'auto' will override any CSS. For instance, I have overflow-x: hidden set on my body. Setting document.body.style.overflow = 'auto' overrides my CSS, and there is no way for me to override react-toolbox's inline style (because inline always wins).